src/main/java/com/ericsson/ei/controller/FailedNotificationControllerImpl.java

```diff
@@ -69,11 +67,12 @@
             queryResponse.setQueryResponseEntity(queryResponseEntity);
             LOGGER.debug("The response is : {}", response.toString());
             if (processMissedNotification.deleteMissedNotification(subscriptionName)) {
-                LOGGER.debug("Missed notification with subscription name {} was successfully removed from database", subscriptionName);
+                LOGGER.debug("Failed notification with subscription name {} was successfully removed from database",
```

Comment:
I think this phrasing is a bit misleading, since the failed notifications does not have a name per se, it's just tied to a specific subscription. Maybe rephrase it to: "Failed notification for subscription {} was successfully removed from database"

Reply:
done